src/modules/75-cf/strings/strings.en.yaml

```
@@ -1539,27 +1539,27 @@ onboarding:
       ```
 
     php: |
-      # Requirements
+      # Quickstart
 
-      Installed <a href="https://www.php.net/">PHP 7.4</a> or a newer version
-      Installed <a href="https://getcomposer.org/">Composer</a>
+      To follow along with our test code sample, make sure you've:
 
-      # Setup
+      - Installed <a href="https://www.php.net/">PHP 7.4</a> or a newer version
+      - Installed <a href="https://getcomposer.org/">Composer</a>
+      - <a href="https://ngdocs.harness.io/article/1j7pdkqh7j-create-a-feature-flag">Created a Feature Flag on the Harness Platform</a> called 'harnessappdemodarkmode'
+      - <a href="https://ngdocs.harness.io/article/1j7pdkqh7j-create-a-feature-flag#step_3_create_an_sdk_key">Created a server SDK key and made a copy of it</a>
+
+
+      # Install
       The first step is to install the SDK as a dependency in your application using Composer.
 
       ```shell
       composer require harness/ff-server-sdk
```

I think this won't actually work. `composer` (by default) will only grab stable versions. You have to tell it what version to grab if you want to get something unstable (like our beta version)
```suggestion
      composer require harness/ff-server-sdk:0.1.0-beta
```

---

Interesting! Should I raise a doc ticket as this is where I got it from?

---

Have addressed here, thank you